### ISSUE TITLE
0.0.5705 fails to build locally

### ISSUE THREAD

**Issue by adborden** [circleci/local-cli#13]
I just updated my circleci and it stopped working with this error.

```
====>> Checkout code
  #!/bin/sh
mkdir -p /home/circleci/project && cp -r /tmp/_circleci_local_build_repo/. /home/circleci/project
cp: cannot stat '/tmp/_circleci_local_build_repo/.': No such file or directory
Error: Exited with code 1
Step failed
Task failed
```

I've seen this on both OSX and Ubuntu.

```
$ circleci version

circleci version: 0.0.5705-58f0e71
Build Agent version: 0.0.5706-58f0e71
built: 2018-05-14T22:35:09+0000
```

**Comment by sgerrand** [comment]
Same issue here.

**Comment by rbngzlv** [comment]
> Running `circleci version` and then `circleci update` should fix the issue. Can you two confirm?

Just updated and the error has gone. Thanks!

**Comment by adborden** [comment]
This was definitely fixed, but I'm running into the same issue. I can't find an image that works.

```
$ circleci version

circleci version: 0.0.5894-01cdb92
Build Agent version: 0.0.5895-01cdb92
built: 2018-05-29T20:07:10+0000

$ docker version
Client:
 Version:           18.06.0-ce
 API version:       1.38
 Go version:        go1.10.3
 Git commit:        0ffa825
 Built:             Wed Jul 18 19:11:02 2018
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.06.0-ce
  API version:      1.38 (minimum version 1.12)
  Go version:       go1.10.3
  Git commit:       0ffa825
  Built:            Wed Jul 18 19:09:05 2018
  OS/Arch:          linux/amd64
  Experimental:     false
```

**Comment by panasyuk** [comment]
Hi there! I experience the same issue on updated CircleCI CLI.
Here's Docker version:
```
Client:
 Version:      17.05.0-ce
 API version:  1.29
 Go version:   go1.7.5
 Git commit:   89658be
 Built:        Thu May  4 22:10:54 2017
 OS/Arch:      linux/amd64

Server:
 Version:      17.05.0-ce
 API version:  1.29 (minimum version 1.12)
 Go version:   go1.7.5
 Git commit:   89658be
 Built:        Thu May  4 22:10:54 2017
 OS/Arch:      linux/amd64
 Experimental: false
```
And CLI version
```
circleci version: 0.1.0
Build Agent version: 0.0.5895-01cdb92
built: 2018-05-29T20:07:10+0000
```

**Comment by Noext** [comment]
hello same issue on mac sierra 10.12:

circleci version: 0.1.0
Build Agent version: 0.0.7725-9681bc36
built: 2018-08-09T15:10:50+0000

docker version
Client:
 Version:      18.03.1-ce
 API version:  1.37
 Go version:   go1.9.5
 Git commit:   9ee9f40
 Built:        Thu Apr 26 07:13:02 2018
 OS/Arch:      darwin/amd64
 Experimental: false
 Orchestrator: swarm

Server:
 Engine:
  Version:      18.03.1-ce
  API version:  1.37 (minimum version 1.12)
  Go version:   go1.9.5
  Git commit:   9ee9f40
  Built:        Thu Apr 26 07:22:38 2018
  OS/Arch:      linux/amd64
  Experimental: true

**Comment by felicianotech** [comment]
@zzak Should these issues be moved to the new repo now?

**Comment by zzak** [comment]
@felicianotech No, we can triage these in place once everyone is able to upgrade to the new CLI

**Comment by JinbinLiang** [comment]
Hi everyone,

I have met this issue with the latest version of circleci local in the environment below. I have tried "circleci version" and "circleci update", but nothing works. Any ideas about that? Thanks.

circleci version
0.1.2191+af75d4d

docker version
Client:
 Version:           18.06.1-ce
 API version:       1.38
 Go version:        go1.10.3
 Git commit:        e68fc7a
 Built:             Tue Aug 21 17:21:31 2018
 OS/Arch:           darwin/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.06.1-ce
  API version:      1.38 (minimum version 1.12)
  Go version:       go1.10.3
  Git commit:       e68fc7a
  Built:            Tue Aug 21 17:29:02 2018
  OS/Arch:          linux/amd64
  Experimental:     true

**Comment by FrodeMMeling** [comment]
Same issue for me.. Maybe something with version 18 of docker that re introduced issue?
